packages/design-system/src/atoms/tooltip/index.web.tsx

```diff
@@ -61,33 +64,39 @@ export const Tooltip: React.FC<TooltipProps> = ({
     useFocus(context),
   ])
 
+  const wrapper = useCallback((render: React.ReactNode) => {
+    return <FloatingPortal>{render}</FloatingPortal>
+  }, [])
+
   return (
     <>
       <div ref={reference} {...getReferenceProps()}>
         {children}
       </div>
-      {(isOpen || open) && (
-        <View
-          style={[
-            containers.borderRadius,
-            containers.shadow,
-            padding.ph16,
-            padding.pv8,
-            // @ts-ignore
-            {
-              zIndex: 999,
-              top: y ?? 0,
-              left: x ?? 0,
-              backgroundColor: variant === 'dark' ? darkNavy : white,
-              position: strategy,
-            },
-          ]}
-          ref={floating}
-          {...getFloatingProps()}
-        >
-          {asTextNode(title, variant === 'dark' ? white : charcoalBlack)}
-        </View>
-      )}
+      <ConditionalWrapper condition={renderInPortal} wrapper={wrapper}>
```

Comment:
Now  we use the prop renderInPortal and the ConditionalWrapper component to render the tooltip inside the FloatingPortal component.